exasol_integration_test_docker_environment/cli/options/test_environment_options.py

```diff
@@ -20,7 +20,9 @@
                  show_default=True,
                  help="""Docker DB Image Name against which the tests should run."""),
     click.option('--create-certificates/--no-create-certificates', default=False,
-                 help="""Creates and injects SSL certificates to the Docker DB container.""")
+                 help="""Creates and injects SSL certificates to the Docker DB container."""),
+    click.option('--additional-db-parameter', type=str, multiple=True,
```

Comment:
This will get quite verbose, if you need to set multiple. Maybe introduce a short version of tbe argument

Reply:
Added a short version